e2e-tests/portal/cypress/e2e/journeys/read-only/fixtures/dealFullyCompleted.js

```diff
@@ -0,0 +1,295 @@
+  previousStatus: 'Draft',
+  comments: [{
+    username: 'bananaman',
+    timestamp: '1984/12/25 00:00:00:001',
+    text: 'Merry Christmas from the 80s',
+  }, {
+    username: 'supergran',
+    timestamp: '1982/12/25 00:00:00:001',
+    text: 'Also Merry Christmas from the 80s',
+  }],
+  mockFacilities: [
+    {
+      type: 'Bond',
+      bondIssuer: 'my issuer',
+      bondType: 'Retention bond',
+      facilityStage: 'Issued',
+      hasBeenIssued: true,
+      ukefGuaranteeInMonths: '12',
+      'requestedCoverStartDate-day': nowDay,
+      'requestedCoverStartDate-month': nowMonth,
+      'requestedCoverStartDate-year': nowYear,
+      'coverEndDate-day': nowPlusMonthDay,
+      'coverEndDate-month': nowPlusMonthMonth,
+      'coverEndDate-year': nowPlusMonthYear,
+      name: '1234567890',
+      bondBeneficiary: 'test',
+      value: '1234',
+      currencySameAsSupplyContractCurrency: 'false',
+      currency: {
+        text: 'GBP - UK Sterling',
+        id: 'GBP',
+      },
+      conversionRate: '100',
+      'conversionRateDate-day': nowMinusDayDay,
+      'conversionRateDate-month': nowMinusDayMonth,
+      'conversionRateDate-year': nowMinusDayYear,
+      riskMarginFee: '12',
+      coveredPercentage: '24',
+      minimumRiskMarginFee: '1',
+      feeType: 'In advance',
+      feeFrequency: 'Quarterly',
+      dayCountBasis: '360',
+      guaranteeFeePayableByBank: '12.345',
+      ukefExposure: '1,234.56',
+    },
+    {
+      type: 'Bond',
+      bondIssuer: 'my issuer',
+      bondType: 'Retention bond',
+      facilityStage: 'Issued',
+      hasBeenIssued: true,
+      ukefGuaranteeInMonths: '12',
+      'requestedCoverStartDate-day': nowDay,
+      'requestedCoverStartDate-month': nowMonth,
+      'requestedCoverStartDate-year': nowYear,
+      'coverEndDate-day': nowPlusMonthDay,
+      'coverEndDate-month': nowPlusMonthMonth,
+      'coverEndDate-year': nowPlusMonthYear,
+      name: '1234567890',
+      bondBeneficiary: 'test',
+      value: '5678',
+      currencySameAsSupplyContractCurrency: 'true',
+      currency: {
+        text: 'CAD - Canadian Dollars',
+        id: 'CAD',
+      },
+      riskMarginFee: '12',
+      coveredPercentage: '24',
+      minimumRiskMarginFee: '1',
+      feeType: 'In arrear',
+      feeFrequency: 'Monthly',
+      dayCountBasis: '360',
+      guaranteeFeePayableByBank: '12.345',
+      ukefExposure: '1,234.56',
+    },
+    {
+      type: 'Loan',
+      facilityStage: 'Conditional',
+      hasBeenIssued: false,
+      ukefGuaranteeInMonths: '12',
+      name: '123456',
+      guaranteeFeePayableByBank: '10.8000',
+      ukefExposure: '2,469,135.60',
+      value: '12345678',
+      currencySameAsSupplyContractCurrency: 'true',
+      interestMarginFee: '12',
+      coveredPercentage: '20',
+      minimumQuarterlyFee: '20',
+      premiumFrequency: 'Monthly',
+      premiumType: 'In advance',
+      dayCountBasis: '365',
+      currency: {
+        text: 'GBP - UK Sterling',
```

Comment:
I believe there are constants of these values and others, can you refer to the constants for various values defined here please.